sdk/formrecognizer/azure-ai-formrecognizer/tests/test_ftc_compose_model.py

```diff
@@ -59,7 +59,8 @@ def test_compose_model_invalid_unlabeled_models_v21(self, client, formrecognizer
 
     @FormRecognizerPreparer()
     @FormTrainingClientPreparer(client_kwargs={"api_version": "2.0"})
-    def test_compose_model_bad_api_version(self, client, formrecognizer_storage_container_sas_url, **kwargs):
```

Comment:
Just double checking the storage container sas url is not used in this test right?

Reply:
Nvm, just checked, thanks for removing it :)